frontend/src/util/helper.js

```diff
@@ -110,23 +110,27 @@ const backendTranslationKeyList = [ // eslint-disable-line no-unused-vars
 
 export const ALL_CONTENT_TYPES = 'html-document,file,thread,folder,comment'
 
-export const sortWorkspaceContents = (a, b) => {
+const naturalCompareItems = (lang, a, b) => {
```

Comment:
If you have a strong preference for the order you suggest, I'll change this, otherwise I'd leave this as is.
I asked myself the same question and I am aware it can feel a bit unatural but I went with this version to allow usage of `bind`.

(this is a common technique in FP languages to order arguments like this - though it is less necessary in Javascript since binding is explicit anyway)

Reply:
ok, i'll change the name, but explicit "item" in the arguments

Reply:
Apart from being shorter, is there a difference between using `bind` and a closure?

Reply:
not really I think

Reply:
ok, I changed the order of the parameters